nextroute/model_next_check.go

```diff
@@ -0,0 +1,195 @@
+package nextroute
+
+import (
+	"context"
+	"time"
+
+	"github.com/nextmv-io/sdk/connect"
+)
+
+// NextCheckModel is the check of a model returning a [NextCheckOutput].
+func NextCheckModel(
+	ctx context.Context,
+	model Model,
+	duration time.Duration,
+	verbosity NextCheckVerbosity,
+) (NextCheckOutput, error) {
+	connect.Connect(con, &nextCheck)
+	return nextCheckModel(ctx, model, duration, verbosity)
+}
+
+// NextCheckSolution is the check of a solution returning a [NextCheckOutput].
+func NextCheckSolution(
+	ctx context.Context,
+	solution Solution,
+	duration time.Duration,
+	verbosity NextCheckVerbosity,
+) (NextCheckOutput, error) {
+	connect.Connect(con, &nextCheck)
+	return nextCheckSolution(ctx, solution, duration, verbosity)
+}
+
+// NextCheckOptions are the options for a check.
+type NextCheckOptions struct {
+	// Duration is the maximum duration allowed for the sanity check to run.
+	Duration  time.Duration `json:"duration" usage:"maximum duration of the sanity check" default:"30s"`
+	Verbosity int           `json:"verbosity" usage:"verbosity of the sanity check, current available options are 0 (off),1 (low), 2 (medium), 3 (high)" default:"0"`
```

Comment:
Verbosity in `mip` was changed to a `string`. I would prefer for that to be the case here as well for consistency.